docs/docs/05-reusable-components.md

```diff
@@ -249,6 +249,15 @@ ReactDOM.render(<HelloMessage name="Sebastian" />, mountNode);
 ```
 
 This simplified component API is intended for components that are pure functions of their props. These components must not retain internal state, do not have backing instances, and do not have the component lifecycle methods. They are pure functional transforms of their input, with zero boilerplate.
+However, you may still specify `.propTypes` and `.defaultProps` by setting them as properties on the function:
```

Comment:
We already have an example of setting proptypes above.  Let's just keep this as a simple note:

 > However, you may still specify `.propTypes` and `.defaultProps` by setting them as properties on the function, just as you would set them on an ES6 class.

And then we can skip the example below, since another example bloats the page and doesn't add a whole lot of value given the equivilant example above.